x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/execution_suspicious_jar_child_process.json

```diff
@@ -0,0 +1,52 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Identifies suspicious child processes of a Java Archive (JAR) file. JAR files may be used to deliver malware in order to evade detection.",
+  "from": "now-9m",
+  "index": [
+    "auditbeat-*",
+    "logs-endpoint.events.*"
+  ],
+  "language": "eql",
+  "license": "Elastic License",
+  "name": "Suspicious JAR Child Process",
```

Comment:
this rule has always been really noisy and never caught TPs from what I remember so I would prefer to drop it and pretend it never existed

Reply:
I believe this one was specifically mentioned by PM as one we did not have for the Mitre evals, hence the reason it was added. I think we can re-evaluate based on telemetry or turn it into a sequence of some sort. 

I also think it could be worthwhile to eliminate `process.executable : /Library/Java/JavaVirtualMachines/*` and its likely to bring the noise down dramatically. 

I'm fine if its cut as well.